ui/src/message_popup/playtest/smith_experience_page.jsx

```diff
@@ -60,6 +61,32 @@ export default React.createClass({
     };
   },
 
+  getNextQuestion(questions, responses) {
+    if (responses.length >= questions.length) {
+      return null;
+    }
+    const question = questions[responses.length];
+
+    // Special case for reviewing past notes when giving feedback.
+    if (question.feedback) {
+      const pastNotes = this.getNotesBeforeFeedbackIndex(questions, responses, responses.length);
+      return {...question, pastNotes};
+    }
+
+    return question;
+  },
+
+  // Returns an array of past notes for review as part of a feedback question.
+  // This looks at all `notes` responses prior to `feedbackQuestionIndex`, only 
+  // looking back until a previous `feedback` question, since there may be multiple
+  // in the same scenario.
+  getNotesBeforeFeedbackIndex(questions, responses, feedbackQuestionIndex) {
```

Comment:
@milsim11 I got mixed up writing this function initially, so wrote a test to verify it's working correctly.  You can see that in `smith_experience_page_test.jsx`, which adds a fake scenario and verifies that the method does what we'd expect.  This is similar to testing manually, but is checked into source control and can run automatically when we make changes and faster too!